cadquery/cq.py

```diff
@@ -2728,6 +2728,66 @@ def _sweep(self, path, multisection=False, makeSolid=True, isFrenet=False,
 
         return Compound.makeCompound(toFuse)
 
+    def interpPlate(self, surf_edges, surf_pts=[], thickness=0, combine=True, clean=True, Degree=3, NbPtsOnCur=15, NbIter=2, Anisotropie=False, Tol2d=0.00001, Tol3d=0.0001, TolAng=0.01, TolCurv=0.1, MaxDeg=8, MaxSegments=9):
+        """
+        Returns a plate surface that is 'thickness' thick, enclosed by 'surf_edge_pts' points,  and going through 'surf_pts' points.
+        
+        :param surf_edges
+        :type 1 surf_edges: list of [x,y,z] float ordered coordinates
```

Comment:
I'm not sure I I understand the intent here

Reply:
Here the idea is that a simple list of points can still be supplied (more convenient sometimes, for example Schwarz D surfaces have 6 straight borders in 6 different planes; in such case the boundary can be defined simply by providing 6 points).
So the user has the choice of defining the surface boundaries as a list of wires or a list of points that will be polygonized.

Reply:
OK, clear.